include/z64actor.h

```diff
@@ -129,7 +129,7 @@ typedef struct Actor {
     /* 0x07D */ u8      floorPolySource; // Complex Poly Surface Source. 0x32 = Scene. related to 0x80/88
     /* 0x07E */ s16     wallPolyRot; // Rotation of the wall poly an actor is touching
     /* 0x080 */ f32     groundY;
-    /* 0x084 */ f32     waterY;
+    /* 0x084 */ f32     yDistToWater;
```

Comment:
I used `To` because it's the y displacement from the actor to the water (e.g. if the actor is above the water plane, yDistToWater will be negative).

The usage is usually `yCoordinateOfWater = this->position.y + this->yDistToWater`

Can we think of a better name?

Reply:
Resolving this based on convo here: https://discord.com/channels/688807550715560050/688851317593997489/793163779709992980

Also pushing the other suggested changes mentioned above
`DistFromLinks` -> `DistToLinks`